_benchmark/user-guide/running-workloads.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: default
+title: Running a workload
+nav_order: 9
+parent: User guide
+---
+
+# Running a workload
+
+Once you understand all of the individual components of a OpenSearch Benchmark (OSB) [workload](add-link-to-understanding-workloads), its time to run your first workload. 
+
+## Step 1: Find the workload name
+
+To find the workload name, use the following command:
```

Comment:
Nit: `To find the workload name, use the following command:` -> `To learn more about the standard workloads that come with OSB, use the following command:`